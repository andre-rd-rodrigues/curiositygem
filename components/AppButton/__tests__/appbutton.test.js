import { render, screen } from "@testing-library/react";
import Button from "../AppButton";
import "@testing-library/jest-dom";

describe("Button", () => {
  it("renders component unchanged", () => {
    const { container } = render(<Button />);

    expect(container).toMatchSnapshot();
  });

  it("renders label from props correctly", () => {
    render(<Button label="read more" />);

    const element = screen.getByText(/read more/i);

    expect(element).toBeInTheDocument();
  });
});
