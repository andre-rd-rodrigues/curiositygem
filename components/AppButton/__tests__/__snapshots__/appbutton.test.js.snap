// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button renders component unchanged 1`] = `
<div>
  <button
    class="container"
  />
</div>
`;
